packages/manager/apps/pci-load-balancer/src/pages/detail/listeners/l7/rules/L7Rules.page.tsx

```diff
@@ -0,0 +1,97 @@
+import { Suspense } from 'react';
+import { Outlet, useParams } from 'react-router-dom';
+import {
+  OsdsBreadcrumb,
+  OsdsIcon,
+  OsdsLink,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { useTranslation } from 'react-i18next';
+import { useProject } from '@ovh-ux/manager-pci-common';
+import { Headers, useProjectUrl } from '@ovh-ux/manager-react-components';
+import { useListener } from '@/api/hook/useListener';
+
+export default function L7RulesPage() {
+  const { t: tPciCommon } = useTranslation('pci-common');
+  const { t: tL7Policies } = useTranslation('octavia-load-balancer-l7');
+  const { t } = useTranslation('l7/rules');
+  const hrefProject = useProjectUrl('public-cloud');
+  const { t: tLoadBalancer } = useTranslation('octavia-load-balancer');
+  const { t: tListener } = useTranslation('octavia-load-balancer-listeners');
+  const { projectId, loadBalancerId, region, listenerId } = useParams();
+  const { data: project } = useProject();
+  const { data: listener } = useListener({
+    projectId,
+    region,
+    loadBalancerId,
+    listenerId,
+  });
+  return (
+    <>
+      <OsdsBreadcrumb
+        items={[
+          {
+            href: hrefProject,
+            label: project.description,
+          },
+          {
+            href: `${hrefProject}/octavia-load-balancer`,
+            label: tLoadBalancer('octavia_load_balancers'),
+          },
+          {
+            href: `${hrefProject}/octavia-load-balancer/${region}/${loadBalancerId}`,
+            label: loadBalancerId,
+          },
+          {
+            href: `${hrefProject}/octavia-load-balancer/${region}/${loadBalancerId}/listeners`,
+            label: tListener('octavia_load_balancer_listeners_title'),
+          },
+          {
+            href: `${hrefProject}/octavia-load-balancer/${region}/${loadBalancerId}/listeners/${listenerId}/l7`,
+            label: tL7Policies('octavia_load_balancer_list_l7_policies_title'),
+          },
+          {
+            label: t('octavia_load_balancer_list_l7_rules_title'),
+          },
+        ]}
+      />
+      <OsdsText
+        className="block mt-8"
+        color={ODS_TEXT_COLOR_INTENT.text}
+        size={ODS_TEXT_SIZE._600}
+        level={ODS_TEXT_LEVEL.heading}
+      >
+        {listener?.name}
+      </OsdsText>
+      <OsdsLink
+        className="mt-8"
+        color={ODS_TEXT_COLOR_INTENT.primary}
+        href={`${hrefProject}/octavia-load-balancer/${region}/${loadBalancerId}/listeners/list`}
```

Comment:
useHref can do the job here `useHref(`../${region}/${loadBalancerId}/listeners/list`)`

Reply:
It will be fixed with the next PR :)